controllers/keystoneendpoint_controller.go

```diff
@@ -309,7 +311,7 @@ func (r *KeystoneEndpointReconciler) reconcileNormal(
 	}
 
 	if !ksSvc.IsReady() {
-		util.LogForObject(helper, fmt.Sprintf("KeystoneService %s not ready, waiting to create endpoints", instance.Spec.ServiceName), instance)
+		logFromCtx.Info(fmt.Sprintf("KeystoneService %s not ready, waiting to create endpoints", instance.Spec.ServiceName), "instance", instance)
```

Comment:
ditto both, drop the instance logging and use structured logging for the KeystoneService